guide/contribute/propose.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: guide
+title: Propose a change
+description: Contribute to the guide by proposing an improvement you've made or fixing an issue
+nav_order: 1
+parent: Contribute to guide
+permalink: /guide/contribute/propose
+image: /assets/images/guide/contribute/contribute-propose-preview.jpg
+---
+
+# Proposing a change
+
+As a free and open-source project, the Bitcoin Design Guide depends on contributions from people all over the internet. Our open-philosophy means that anyone can be a part of the community and improve the guide. No matter if you’re an experienced developer looking to improve the code, a designer looking to enhance visuals, or you just want to fix a typo — your contributions are welcomed.
+
+Anyone can make a difference and become a contributor. This page explains how to make your first contribution by fixing an issue or making an improvement.
+
+## GitHub
+
+The entire source code and content of the guide are hosted in a cloud, [on GitHub](https://github.com/BitcoinDesign/Guide/). GitHub is a collaborative platform which at its core has a distributed version control system called [Git](https://git-scm.com/). GitHub doesn't just host the code, it also allows smooth collaboration and interaction between contributors.
+
+{% include youtube.html id="0vMBdtOnHJM" %}
+
+### Prerequisites
+
+While advanced users can interact with GitHub through a command line, this guide focuses on showing how to work with Git locally, by using the following software:
+
+1. Create a [GitHub account](https://github.com/)
+2. [Download and install GitHub Desktop](https://desktop.github.com/) (macOS and Windows) / [Linux](https://github.com/shiftkey/desktop)
+3. Download and install text editor ([Sublime Text](https://www.sublimetext.com/), [Atom](https://atom.io/), etc)
+
+### Identifying what to fix
+
+If you're looking to help, but not sure where to begin:
+
+- Check issues labeled as [Good first issue](https://github.com/BitcoinDesign/Guide/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```

Comment:
I've probably missed it somewhere but have we spoke about/defined what a good first issue is?

Reply:
The `Good first issue` is something quite standard on GitHub. It's usually an issue that's good for first-time contributors. Usually a small fix and easy to understand, actionable issue.

Reply:
+ self contained tasks - they have little to no dependencies.